pywr-core/src/recorders/aggregator.rs

```diff
@@ -42,14 +42,10 @@ impl AggregationPeriod {
         while current_date < end_date {
             // This should be safe to unwrap as it will always create a valid date unless
             // we are at the limit of dates that are representable.
```

Comment:
by the time crate?

Reply:
Actually, this comment is old and should be moved to the `start_of_next_period` function.